### DISCUSSION TITLE
Mac OS standard key bindings

### DISCUSSION THREAD

**Discussion by fullstackplus** [wezterm/wezterm#6404] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Hi, 

I'm on Mac OS using Wezterm and only the parentheses key binding works (shift+parens). Square brackets (option+parens) don't work and neither do curly brackets (option+shift+parens). 

I thought these key bindings were universal across apps — how to I add them to Wezterm? Thanx.

**Comment by Blutsh** [comment, marked as answer]
Hello,

In order to make it work you'll need to add these lines in your [wezterm dot file ](https://wezfurlong.org/wezterm/config/files.html)( ~/.wezterm.lua).

```
config.send_composed_key_when_left_alt_is_pressed = true
config.send_composed_key_when_right_alt_is_pressed = true

```

**Reply by fullstackplus** [reply]
Thank you @Blutsh!
